### DISCUSSION TITLE
Xcode cannot recognise generated certificates with multiple apps

### DISCUSSION THREAD

**Discussion by paulosllvn** [fastlane/fastlane#19805] [Q&A]
We have two applications under the same apple developer account and fastlane is utilised to build both of these applications.

Both apps have their own separate repo and both have separate repos to hold their profiles and certs.

When fastlane generates new certs and profiles for each app, it generates an identical certificate for both applications and adds them to the keychain. This seems to cause Xcode to get confused and not pick up on the correct certificate for one of the projects, resulting in one of the projects working flawlessly and the other project giving this error:

![image](https://user-images.githubusercontent.com/43392498/149445410-86add608-3987-4a6e-8b45-6a2de68c7d71.png)

Any ideas on how to resolve this without creating a new developer account and splitting out each project?

[a comment by an automated account is not shown]

**Comment by lucgrabowski** [comment]
Hi, I see 2 possibilities:
1. You could have 1 certificate and profiles for both apps in 1 repository, even if both apps have separate repositories.
2. You could create certificate in one repository and do manual work: export certificate manually and import it in the other repository

If it's possible, I would go for (1)

**Comment by paulosllvn** [comment]
> Hi, I see 2 possibilities:
> 
> 1. You could have 1 certificate and profiles for both apps in 1 repository, even if both apps have separate repositories.
> 2. You could create certificate in one repository and do manual work: export certificate manually and import it in the other repository
> 
> If it's possible, I would go for (1)

Thanks :) This did the trick!
